files/image_config/monit/memory_checker

```diff
@@ -59,6 +69,81 @@ def get_command_result(command):
 
     return command_stdout.strip()
 
+def get_container_id(container_name):
+    """Gets full container ID of the specified container
+    Args:
+        container_name: A string indicates the name of specified container.
+    Returns:
+        container_id: A string indicates the full ID of specified container.
+    """
+    container_id = ""
+
+    get_container_info_cmd = "docker ps --no-trunc"
+    command_stdout = get_command_result(get_container_info_cmd)
+
+    for line in command_stdout.splitlines():
+        if container_name in line:
+            container_id = line.split()[0].strip()
+            break
+
+    if not container_id:
+        syslog.syslog(syslog.LOG_ERR, ERROR_CONTAINER_ID_NOT_FOUND.format(container_name))
```

Comment:
Minor suggestion: Can we refactor this log usage to be a common function, like

def log_and_exit(error_message, exit_code):
    syslog.syslog(syslog.LOG_ERR, error_message)
    sys.exit(exit_code)

Also, given exit_code constants in top of script?

Reply:
In current code not all the syslog require exit actually, so we may keep it first as error codes have been defined on the top.